### PR TITLE
Added an example of data with error bars

### PR DESCRIPTION
**Pull request by dhuppenkothen**
This is a really simple example, but almost all of my data is of this type and it took me a bit to figure out `transform_calculate`, so I thought it might be useful for someone else to see. 

This is kind of a simpler version of the [errorbar example](https://altair-viz.github.io/gallery/error_bars_with_ci.html), where data points and uncertainties don't have to be calculated from a distribution, but are given as (x, y, yerr).

### CODE REVIEW

**Comment by jakevdp** [comment]
Thanks!

I'd put that in the "Scatter Plots" category... the "Simple Charts" category is mainly for simple examples without layering, transforms, etc.

**Comment by dhuppenkothen** [comment]
Wherever you think it fits best. Is there something I should change in the files or the description to make that work?

File: altair/vegalite/v2/examples/simple_scatter_with_errorbars.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""
+Simple Scatter Plot with Errorbars
+----------------------------------
+
+A simple scatter plot of a data set with errorbars.
+
+"""
+# category: simple charts
+
```
Comment (jakevdp) [inline altair/vegalite/v2/examples/simple_scatter_with_errorbars.py]: Change this to ``# category: scatter plots``

**Comment by jakevdp** [comment]
Looks good, thanks!

**Pull request merged**